docs/workflows/build-pages-with-storybook.md

```diff
@@ -117,13 +117,75 @@ module.exports = {
   // your Storybook configuration
 
   webpackFinal: config => {
-    config.resolve.alias.fetch = '../__mocks__/fetch.js'
+    config.resolve.alias['isomorphic-fetch'] = require.resolve('../__mocks__/isomorphic-fetch.js');
     return config;
    }
 };
 ```
 
-You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story.
+You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story. An example of how that might look:
+
+```js
+// __mocks__/isomorphic-fetch.js
+let nextJson;
+export default async function fetch() {
+  if (nextJson) {
+    return {
+      json: () => nextJson,
+    };
+  }
+  nextJson = null;
+}
+
+export function decorator(story, { parameters }) {
+  if (parameters && parameters.fetch) {
+    nextJson = parameters.fetch.json;
+  }
+  return story();
+}
+```
+
+You would then wire it up like so:
+```js
+// .storybook/preview.js
+import { decorator } from '../__mocks/isomorphic-fetch';
+
+// Add the decorator to all stories
+export const decorators = [decorator];
+```
```

Comment:
@tmeasday we could probably improve this a bit we could say:

"Add the mock you've just implemented to your `storybook/preview.js` (if you don't have it already, you'll need to create the file)."

WDYT?